package.json

```diff
@@ -64,6 +64,7 @@
     "grunt-jscs": "2.1.0",
     "grunt-mocha-cli": "1.14.0",
     "hawk": "3.1.0",
+    "halfpenny": "*",
```

Comment:
I like the name! 

Let's take a moment to consider the development workflow in this situation: 
1. Developer adds new route to API
1. Developer adds new methods to halfpenny client to utilize new route
1. Developer publishes halfpenny client with new methods
1. Developer installs latest version of halfpenny in order to run tests on new API routes.

I see this workflow could lead to problems because halfpenny needs to be updated and published first in an untested state in order to test new API changes. Additionally, there is no way for different branches of the nodeapi to reference different versions of halfpenny (unless we pin to specific versions).

Here are a couple of solutions:
1. Pin halfpenny to a specific version, then change that version number to keep it in-check with the nodeapi.
  1. Pros: uses versioning
  1. Cons: still requires that a new version of halfpenny be published before testing new nodeapi features.
1. Use github URLs with branch names for the halfpenny package instead of NPM versions.
  1. Pros: allows each branch of the nodeapi to reference the corresponding branch of halfpenny
  1. Cons: 
    1. would require a pre-install step to append the current branch name to the halfpenny github URL
    1. If a developer forgot to reinstall after changing branches, it could cause serious problems.
    1. When creating feature branches, a feature branch of halfpenny would also need to be made in order for tests to run.
1. Treat halfpenny as a peer of nodeapi (like coins_core and micis). All references to halfpenny will be to its expected absolute path (e.g. _/coins/halfpenny_).
  1. Pros: 
    1. halfpenny will only need to be branched when necessary.
    1. halfpenny can go through the same dev/release process as other COINS repos
    1. halfpenny does not need to ship with the nodeapi to staging and production.
  1. Cons:
    1.  developer must keep checked out branches of halfpenny and nodeapi in sync (just like they do with coins_core, micis, cas, etc now...)


What do you think? My preference lies with the third option currently.